source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -38,14 +38,19 @@ AndroidContext: class extends OpenGLContext {
 	}
 	createGpuImage: override func (rasterImage: RasterImage) -> GpuImage {
 		result: GpuImage
-		if (rasterImage instanceOf?(GraphicBufferYuv420Semiplanar)) {
-			graphicBufferImage := rasterImage as GraphicBufferYuv420Semiplanar
-			rgba := graphicBufferImage toRgba(this)
-			result = this unpackBgraToYuv420Semiplanar(rgba, rasterImage size)
-			rgba free()
+		version (graphicbufferupload) {
```

Comment:
Debug flags should be named with the prefix `debug` and then what's it's debugging. In this case something like `debugGraphicbufferupload` or similar.